packages/lesswrong/components/posts/SuggestCurated.tsx

```diff
@@ -45,7 +45,8 @@ const SuggestCurated = ({post}: {
       !post.curatedDate &&
       !post.reviewForCuratedUserId &&
       forumTypeSetting.get() !== 'AlignmentForum' &&
-      userCanDo(currentUser, "posts.moderate.all"))
+      (userCanDo(currentUser, "posts.moderate.all")
+        || userIsMemberOf(currentUser, 'canSuggestCuration')))
```

Comment:
```suggestion
      (userCanDo(currentUser, "posts.moderate.all") || 
      userIsMemberOf(currentUser, 'canSuggestCuration')))
```

matching previous lines